### ISSUE TITLE
nand flash

### ISSUE THREAD

**Issue by v6760hu** [AndrejChoo/USB-Programmer-parallel-flash-and-eeprom#2]
https://github.com/bbogush/nand_programmer
Can I read nand flash

**Comment by AndrejChoo** [comment]
Hi. NAND flash not supported, only parallel flash and eeprom.

**Comment by v6760hu** [comment]
Using pin stm32f407vet6 i2c 93 and 92 serial ports to communicate with the upper computer will be faster. Because I saw Nano adopting this fast read and write solution.

**Comment by AndrejChoo** [comment]
I do not think so. My device uses a hardware USB as a virtual com port, it uses bulk endpoints, and they are the fastest.
